docs/buckets.rst

```diff
@@ -0,0 +1,141 @@
+Buckets
+#######
+
+.. _buckets:
+
+:ref:`Buckets <buckets>` enables the creation of collections handled by
+a group of ref:`User identifiers <user-identifiers>`.
+
+All collections are created in a bucket. By default, it uses the connected
+user's bucket (bucket only the connected user has access to).
```

Comment:
By default, the bucket of the current user is used.

(the second part between parenthesis could not be processed by my brain at this time of the day)